portfolio/src/main/java/com/google/sps/servlets/ServeURLServlet.java

```diff
@@ -0,0 +1,26 @@
+import java.io.IOException;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.annotation.WebServlet;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+
+/** This class provides support for the BlobStore Service, we store files as
+    blobkeys, so to acces them on the web page we need to generate a serving
+    url, which will be returned in the response of the GET request. 
+    This fixes the problem with ImageService, since it does not use that anymore.*/
```

Comment:
What problem with ImageService? This part of the comment would probably make sense to put in the PR, rather than in the code, since it's requires a bit of context regarding the previous state of the code.

Reply:
Well to be honest I don't know, apparently when you deploy it on the page, the way they exemplified us to get a serving url does not work, it throws ImageServiceFailureException or something similar, so I made a bit of research on another method to generate a serving url for a blob.